src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[]{new Bulldozer(), new Excavator(), new Truck()};
+        for (Machine mac : machines) {
```

Comment:
please do not cut the variable names, use the whole words according the clean code recommentations